src/main/resources/log4j.properties

```diff
@@ -0,0 +1,11 @@
+log4j.rootLogger=DEBUG,console,file
+
+log4j.appender.console=org.apache.log4j.ConsoleAppender
+log4j.appender.console.Target=System.out
+log4j.appender.console.layout=org.apache.log4j.PatternLayout
+
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=chain\\log.log
```

Comment:
Path from previos review was better :))

Reply:
Use log.log without any additional folders. It will be saved on root folder of your project.